### PR TITLE
refs #21350 - allow config of TLS version on 8443

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by ehelms** [comment]
We may also need to set this on `pulp::crane`.

**Comment by stbenjam** [comment]
@ehelms Updated

**Comment by ekohl** [comment]
> We may also need to set this on pulp::crane.

Since that defaults to undef it can already be overridden on a global apache level.

File: manifests/init.pp [hunk 1]
```diff
@@ -127,6 +131,7 @@
       key      => $certs::apache::apache_key,
       ca_cert  => $certs::ca_cert,
       data_dir => '/var/lib/pulp/published/docker/v2/app',
+      ssl_protocol => $ssl_protocol,
```
Comment (ekohl) [inline manifests/init.pp]: Alignment is off

Comment (stbenjam) [reply manifests/init.pp]: Fixed

**Pull request merged**